### ISSUE TITLE
export was not found in 'firebase/app'

### ISSUE THREAD

**Issue by steltz** [nuxt-community/firebase-module#349]
I'm getting the following warning messages in the console:

"export 'apps' (imported as 'firebase') was not found in 'firebase/app'
"export 'auth' (imported as 'firebase') was not found in 'firebase/app'
"export 'initializeApp' (imported as 'firebase') was not found in 'firebase/app'

The app won't load in browser with the following error:

`TypeError: Cannot read property 'length' of undefined`

I believe its bc Firebase isn't initialized correctly, anyone have any pointers?

Here is my nuxt.config.js:

```
  modules: [
    // https://go.nuxtjs.dev/axios
    '@nuxtjs/axios',
    // https://go.nuxtjs.dev/pwa
    '@nuxtjs/pwa',
    // https://go.nuxtjs.dev/content
    '@nuxt/content',
    '@nuxtjs/firebase'
  ],

  firebase: {
    config: {
     ...
    },
    services: {
      auth: true
    }
  },
```

**Comment by FougereBle** [comment]
Try to install firebase@^7.17.1 package.
`npm install firebase@^7.17.1`

**Comment by HamoBoker** [comment]
this issue happened to me also
firebase latest version 8.0.0 causing this problem.
i switch to version 7.24.0 and everything works correctly

thanks to @FougereBle

**Comment by halilduygulu** [comment]
We need new release of this library to use firebase 8 version, imports are breaking change. 
https://firebase.google.com/support/release-notes/js

Any timeline for this? This firebase release supposed to lower bundle size a bit, I am curious to try it.

**Comment by lupas** [comment]
Hey guys

Will try to release a new version within the next 2 days to fix this - hang in there! Until then the only solution is to use Firebase version < 8.0.0.

**Comment by lupas** [comment]
Fixed in https://github.com/nuxt-community/firebase-module/commit/4ae48b51bc35f007c89bcc46febeded30f288893
- will be released in v7.0.0 in the coming days.
